### PR TITLE
Update all HF tests to use new runner

### PR DESCRIPTION
**Pull request by jysohn23**

### CODE REVIEW

File: tests/pytorch/nightly/hf-glue.libsonnet [hunk 1]
```diff
@@ -209,11 +213,11 @@ local utils = import "templates/utils.libsonnet";
     accelerator: tpus.v3_8,
   },
   configs: [
-    hf_glue + v3_8 + bert_base_cased + timeouts.Hours(1),
-    hf_glue + v2_8 + bert_base_cased + timeouts.Hours(1),
-    hf_glue + v3_8 + xlnet_large_cased + timeouts.Hours(3),
-    hf_glue + v3_8 + roberta_large + timeouts.Hours(3),
-    hf_glue + v3_8 + xlm_mlm_en_2048 + timeouts.Hours(4),
-    hf_glue + v3_8 + distilbert_base_uncased + timeouts.Hours(1),
+    hf_glue + v3_8 + bert_base_cased + timeouts.Hours(2),
```
Comment (taylanbil) [inline tests/pytorch/nightly/hf-glue.libsonnet:216]: why the increase? are you seeing slowdowns?

Comment (jysohn23) [reply tests/pytorch/nightly/hf-glue.libsonnet:216]: Yeah mainly at the dataset setup stage, which was refactored by HF. But training time wise no, no slowdown afaict.

File: tests/pytorch/nightly/hf-glue.libsonnet [hunk 2]
```diff
@@ -22,46 +22,50 @@ local utils = import "templates/utils.libsonnet";
...
-        --eval_batch_size 32
-      ||| % {common: command_common}
+        %(common)s --model_name_or_path bert-base-cased \
+        --per_gpu_train_batch_size 128 \
+        --per_gpu_eval_batch_size 128
+        %(common_copy)s
+      ||| % {common: command_common, common_copy: command_copy_metrics}
     ),
     regressionTestConfig+: {
       alert_for_failed_jobs: false,
```
Comment (zcain117) [inline tests/pytorch/nightly/hf-glue.libsonnet:58]: if you think these tests will pass, you can remove all instances of this flag for these tests (it defaults to `true`)

Comment (jysohn23) [reply tests/pytorch/nightly/hf-glue.libsonnet:58]: Which flag are you referring to?

Comment (zcain117) [reply tests/pytorch/nightly/hf-glue.libsonnet:58]: alert_for_failed_jobs

**Pull request merged**